MetaTempUnit/Hardware/library/KelvinVR/src/KelvinVR.cpp

```diff
@@ -292,7 +346,7 @@ void Kelvin::processCommand(const String &command)
   }
   case hash("HEAT"):
   {
-    int intensity = 255;
+    int intensity = MAX_INTENSITY;
     if (command.length() > 5)
```

Comment:
[nit] Instead of number there suppose to be comparing with size of "HEAT" string

Reply:
it's the fastest way for now

Reply:
The length for now is working this way because otherwise another library required just for this case